Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/configuration/update/CCLConfigurationUpdateScheduler.kt

```diff
@@ -48,14 +47,13 @@ class CCLConfigurationUpdateScheduler @Inject constructor(
     }
 
     private fun buildWorkRequest(): PeriodicWorkRequest {
-        val constraints = Constraints.Builder()
```

Comment:
We shouldn't only run the worker when we have network connection, since the dccwallet recalculation should run daily even if we can't get the config from the network.